app/features/routes.py

```diff
@@ -0,0 +1,75 @@
+from app import db
+from app.features import bp
+from flask import render_template, request
+from flask_login import login_required
+from app.dummy_feature.models import Sensor, sensorFeature, SensorType, TemperatureData, Feature, Room
+import random
+import json
+import datetime
+
+
+# Feature selection page
+@bp.route('/feature_select')
+@login_required
+def feature_selector():
+    features = Feature.query.all()
+    if len(features) <= 0:
+        sensor = Sensor.query.get(1)
```

Comment:
Variable "sensor" is never checked, if sensor is null, python code will be crash.